intake/catalog/local.py

```diff
@@ -512,7 +512,11 @@ def register_plugin_module(mod):
         if k:
             if isinstance(k, (list, tuple)):
                 k = k[0]
-            global_registry[k] = v
+            # load_plugins_from_module can employ imp.load_source in which case
+            # we will re-exec the code and get a different type each time, so
+            # we have to tolerate overwriting here.
+            uses_load_source = mod.endswith('.py')
```

Comment:
Yep, suggest raising an error for this case and dropping in `register_driver`. In hindsight, it's quite a security hole to automatically execute code just because you opened a catalogue, right? Also, it won't work for remote catalogues.

Reply:
Both good points. Will do.

Reply:
Done in 6e9b2b1c2110b3bfa55a95bb5496b43d82b29e84